ToolBox/BovespaConverter/config.json

```diff
@@ -0,0 +1,28 @@
+﻿{
+    //----------------------------------------
+    // Configuration for the Bovespa Converter
+    //----------------------------------------
+
+    // Input folder in which zip raw files are saved
+    "input-folder": "C:\\Users\\Alexandre\\Documents\\Data",
+
+    // Output folder in which files will be saved (usually the Lean data folder)
+    "output-folder": "C:\\Users\\Alexandre\\Documents\\GitHub\\Lean\\Data",
```

Comment:
The input folder can be anything right - is it just a cache for ftp download? So to make it work in anyone's computer we could change it to "./cache" with a Directory.CreateDirectory call if it doesn't exist.

The output folder could be relative to the toolbox bin directory - "../../Data" so it will save data into the default data directory for LEAN.

Very cool how you made factor files by the way. We should probably extract that code into its own function -- e.g. move it into the LEAN data writer and make it accept generic input to generate the factors.